docs/home/contribute/participating.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Participating in SIG-DOCS
+---
+
+{% capture overview %}
+
+SIG-DOCS is one of the [special interest groups](https://github.com/kubernetes/community/blob/master/sig-list.md) within the Kubernetes project, focused on writing, updating, and maintaining the documentation for the project as a whole.
+
+{% endcapture %}
+
+{% capture body %}
+
+Everyone is encouraged to provide content and reviews to support the Kubernetes project documentation. Anyone may open a pull request, and anyone is welcome to provide their opinions and feedback on those pull requests.
+
+Within the Kubernetes project, you may also become a member, reviewer, or approver. These levels confer additional privileges when it comes to approving and committing changes. You may also want to read [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for more details on how membership works within the Kubernetes community.
+
+## Roles and Responsibilities
+
+- Members
+
+Anyone who is a member of the Kubernetes organization can provide a review for a pull request. The SIG-DOCS team will frequently request a review for technical accuracy, and if you are a member, then you can indicate your approval by adding a comment of `/lgtm` to a pull request.
+
+- Reviewers
+
+Reviewers are individuals who have volunteered to provide their time to review documentation pull requests. You become a reviewer by adding your github handle to the [OWNERS file](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md) within the [website repo](https://github.com/kubernetes/website).
+
+Our automation will reviewers to use for reviews on pull requests, and a contributor may request your review with an `/assign [your_github_handle]` comment in the pull request. To indicate your approval of a pull request, a reviewer should add an `/approve` comment to the pull request. A reviewer may also use add a `/hold` comment on the pull request to prevent the pull request from being merged. Another reviewer or approver may also remove that hold with the comment `/hold cancel`.
```

Comment:
"our automation will add reviewers"?
Also -- I'm confused, I thought /lgtm was the bot command for reviewers, and that /approve meant something to the bot only if an approver added it ...

Reply:
This paragraph could be clearer. Maybe something like:

> Our automation assigns reviewers to pull requests, and contributors can request your review with a comment on the pull request: `/assign [your_github_handle]`. To indicate that a pull request requires no further changes, a reviewer adds a comment to the pull request: `/lgtm`. 
> 
> Approvers can indicate their approval with a comment: `/approve`. 
> 
> Reviewers and approvers can add a `/hold` comment to prevent the pull request from being merged. Another reviewer or approver can remove a hold with the comment `/hold cancel`.

Reply:
we'd better clarify the usage of labels. my current understanding is that for reviewers and approvers, `/lgtm` means "technology approval", `/approve` means "documentation reads okay". Besides these, for approvers, `/approve` triggers a merge.
If I am misunderstandin